dot-net/Centroid/Config.cs

```diff
@@ -78,7 +78,23 @@ public override bool TryConvert(ConvertBinder binder, out object result)
 
             try
             {
-                result = Convert.ChangeType(RawConfig, binder.Type);
+                JToken rawConfig = RawConfig;
+                result = rawConfig.ToObject(binder.Type);
+                //ensure at least one property in the config bound to the type
```

Comment:
Yeah, some testing reveals that this doesn't work well for the purpose I need it to today: to `IEnumerable<T>` -- needs to be able to be recursive.

This was just an arbitrary way that I concocted to try to check to see if the result actually bound to anything.  If this logic isn't here, then the fail test does return the object, just with the default values.